app_python/main.py

```diff
@@ -16,10 +17,33 @@
 @bp.route('/')
 def main():
     time_msc = datetime.now(ZoneInfo("Europe/Moscow")).time()
+    time_text = time_msc.isoformat(timespec='seconds')
+    
+    write_log = Process(target=saveLog, args=(f"{time_text} - {request.remote_addr}\n",), daemon=True)
+    write_log.start()
+
     return ("Moscow time: " +
-        time_msc.isoformat(timespec='seconds') +
+        time_text +
         "<br><br>(the time is actual for the last webpage load)")
 
+def saveLog(log_str):
+    with open("/app/visits/visits.txt", "a+") as fo:
+        fo.write(log_str)
+
+@bp.route('/visits')
+def visits():
+    web_content = "History of visits at timestamps and requester's IP:<br>"
+
+    try:
+        with open("/app/visits/visits.txt", "r") as fo:   
+            file_text = fo.read().replace('\n', '<br>')
```

Comment:
funny, tho not sure if it's really necessary to `.replace` every time if you can just write `<br>` to files ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)

but :trollface:  if it works don't touch it ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)

Reply:
Log file be like:<br>
Yes<br>

Reply:
log file goes brrrrrrrrrrr